### PR TITLE
Add a Keycloak Benchmark CI runner script, to execute simulations on a CI System (fixes #134)

### LINKED ISSUE #134: add a keycloak benchmark ci runner shell script to run simulations on a remote ci agent
### Description

add a keycloak benchmark ci runner shell script to run simulations on a remote ci agent

### Discussion

_No response_

### Motivation

To be able to scale the number of runs that can be executed using the existing framework

### Details

This particular script would be designed with a specific Jenkins CI system in mind, however, similar approach can be adopted to any other CI systems around.

### PR DESCRIPTION
**Pull request by kami619**
This script assumes, it will intake certain environment and parametrized variables coming in from a CI system job (ex. Jenkins)

The script automates a bunch of actions needed, such as

  - Reset the Keycloak K8s setup using minikube

  - SET all the needed environment variables and configurations for the simulation and execute the simulation

  - Archive the Gatling results, create the custom report and store all the Archived resutls, reports in a central location

### CODE REVIEW

File: kcb-ci-runner.sh
Comment (ahus1) [inline kcb-ci-runner.sh:46]: We're running in CI here, please add a "-B" to the Maven command line to minimize log output.

File: kcb-ci-runner.sh
Comment (ahus1) [inline kcb-ci-runner.sh:60]: Please avoid "difficult" characters in the folder name, it could be as constrained as "20220712T122009Z"

**Review by ahus1 — COMMENTED** [review]
Thanks for the PR! It looks like what we need here, see some minor comments below. As discussed in the previous call, it would be nice to have all contents of the report folder as one ZIP file so I can download the ZIP directly when browsing the folders via my web browser.

File: kcb-ci-runner.sh
Comment (kami619) [reply kcb-ci-runner.sh]: fixed it

File: kcb-ci-runner.sh
Comment (kami619) [reply kcb-ci-runner.sh]: sure thing, added it

File: kcb-ci-runner.sh
Comment (kami619) [reply kcb-ci-runner.sh]: makes sense, made it easier to read now

**Comment by kami619** [comment]
@ahus1 i have implemented the feedback review, please have a look.

**Review by ahus1 — APPROVED** [review]
This looks good, thank you for this script. I added one more comment that I missed on the first review. Once this has been updated, please merge this PR right away without another review. Thanks!

File: kcb-ci-runner.sh
Comment (kami619) [reply kcb-ci-runner.sh]: thats true, fixed it.

**Pull request merged**
